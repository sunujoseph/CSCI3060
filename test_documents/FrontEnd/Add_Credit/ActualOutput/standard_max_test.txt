Enter Username: 
Enter Amount To Add To Account: 
Error: Maximum Amount To Add Is 1000.00
Enter Amount To Add To Account: 