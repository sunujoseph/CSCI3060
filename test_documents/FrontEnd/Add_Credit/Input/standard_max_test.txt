login
userFS
addcredit
1001
