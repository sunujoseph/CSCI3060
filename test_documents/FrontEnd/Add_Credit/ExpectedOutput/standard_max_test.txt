<<<<<<< HEAD
Enter user name: 
Enter amount of credit to add to your account: 
Cannot add more than $1000.00 to your account at one time.
Logged out.
=======
Enter Username: 
Enter Amount To Add To Account: 
Error: Maximum Amount To Add Is 1000.00
Enter Amount To Add To Account: 
>>>>>>> 9c2a426d2fb208ea0ac3da5fb156edc1d3a04f44
